config.w32

```diff
@@ -80,7 +86,7 @@ if (PHP_MONGODB != "no") {
     BSON_HAVE_TIMESPEC: 0,
     BSON_EXTRA_ALIGN: 0,
     BSON_HAVE_SYSCALL_TID: 0,
-    BSON_HAVE_DECIMAL128: 0,
```

Comment:
The trailing comma did raise an error. I'm not sure why this wasn't an issue before, but I noticed it after updating my php-sdk sources and deps (now 5.6.25 and 7.0.10).